ql/src/java/org/apache/hadoop/hive/ql/ddl/table/create/like/CreateTableLikeOperation.java

```diff
@@ -148,13 +151,20 @@ private void setUserSpecifiedLocation(Table table) {
 
   private void setTableParameters(Table tbl, Map<String, String> originalProperties) throws HiveException {
     // With Hive-25813, we'll not copy over table properties from the source.
-    // CTLT should should copy column schema but not table properties. It is also consistent
+    // CTLT should copy column schema but not table properties. It is also consistent
     // with other query engines like mysql, redshift.
     originalProperties.putAll(tbl.getParameters());
     tbl.getParameters().clear();
     if (desc.getTblProps() != null) {
```

Comment:
The `desc.getTblProps()` could be overwritten by the retain properties, is there any reason to do this?